src/features/notice/ui/NoticeList/NoticeList.tsx

```
@@ -0,0 +1,236 @@
+
+	const handleDelete = (id: string) => {
+		try {
+			removeNotice(id);
+			message.success('공지사항이 성공적으로 삭제되었습니다.');
+		} catch (error) {
+			console.error('공지사항 삭제 실패:', error);
+			message.error('공지사항 삭제에 실패했습니다.');
+		}
+	};
+
+	const handleDragEnd = (event: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)